CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Guidelines for Pull Request
+
+* Avoid pull requests that :
+      *are automated or scripted
+      * that are plagarized from someone else's branch
+  * Do not spam
+  * Project maintainer's decision on validity of PR is final.
+
+  For additional guidelines, refer to [participation rules](https://hacktoberfest.digitalocean.com/details#rules)
+  
```

Comment:
these rules are added, move the PR related content from readme to docs folder.

Reply:
d2f8994 not sure if this is what you meant by moving the content to docs

Reply:
create a new file in docs folder, which has instructions on how to contribute